source/developers-guide/plugin-configuration/index.md

```diff
@@ -242,6 +242,35 @@ To make your selectfield / combobox multiple selectable, you need to create the
 </element>
 ```
 
+### Remote combobox with own store
+Define and create your own ExtJs data store for the backend Plugin configuration
+
+```xml
+<element scope="shop" type="select">
+           ...
+            <store><![CDATA[Ext.define('YourPrefix.YourName.Form', {
+    extend: 'Ext.data.Store',
+    fields: [
+        { name:'id', type: 'int' },
+        { name:'name', type: 'string' }
+    ],
+    autoLoad: true,
+    pageSize: 25,
+    proxy: {
+        type: 'ajax',
+        url: window.location.href.substr(0, window.location.href.indexOf('backend')) + 'backend/YourController/YourAction',
```

Comment:
an example of the response from the controller would be nice

Reply:
Done